docs/user/dashboard/lens-advanced.asciidoc

```diff
@@ -0,0 +1,259 @@
+[[create-a-dashboard-of-panels-with-ecommerce-data]]
+== Tutorial: Create a dashboard of panels with ecommerce sales data
+
+You collected sales data from your store, and you want to visualize and analyze the data on a dashboard. 
+To create dashboard panels of the data, open the *Lens* visualization builder, then  
+create the visualization panels that best display the data.
+
+When you've completed the tutorial, you'll have a dashboard that provides you with a complete overview of your ecommerce sales data.
+
+[role="screenshot"]
+image::images/lens_advanced_result.png[Dashboard view]
+
+[discrete]
+[[add-the-data-and-create-the-dashboard]]
+=== Add the data and create the dashboard
+
+To create visualizations of the data from your store, add the data set, then create the dashboard.
+
+. From the {kib} *Home* page, click *Try our sample data*.
+
+. From *Sample eCommerce orders*, click *Add data*.
+
+. Open the main menu, then click *Dashboard*.
+
+. On the *Dashboards* page, click *Create dashboard*.
+
+[float]
+[[open-and-set-up-lens]]
+=== Open and set up Lens
+
+Open the *Lens* editor, then make sure the correct fields appear.
+
+. From the dashboard, click *Create panel*.
+
+. On the *New visualization* window, click *Lens*.
++
+[role="screenshot"]
+image::images/lens_end_to_end_1_1.png[New visualization popover]
+
+. Make sure the *kibana_sample_data_ecommerce_* index appears.
++
+The list of fields that appear depend on the <<index-patterns,index pattern>>, <<set-time-filter,time filter>>, and field filters. 
+
+[discrete]
+[[view-the-number-of-transactions-per-day]]
+=== View the number of transactions per hour
+
+To determine the number of orders made per hour, create a bar chart, then add it to the dashboard.
+
+. Set the <<set-time-filter,time filter>> to *Last 30 days*.
+
+. From the *Available fields* list, drag and drop *Records* to the visualization builder.
++
+[role="screenshot"]
+image::images/lens_advanced_1_1.png[Added records to the workspace]
+
+. Change the *Vertical axis* title and display the number of orders per day. 
+
+.. From the editor, click *Count of Records*.
+
+.. In the *Display name* field, enter `Number of orders`.
+
+.. Click *Add advanced options > Normalize by unit*.
+
+.. From the *Normalize by unit* dropdown, select *per hour*, then click *Close*.
+
+. To hide the *Horizontal axis* label, open the *Bottom Axis* menu, then deselect *Show*.
++
+[role="screenshot"]
+image::images/lens_advanced_1_1_2.png[Bottom axis menu]
++
+You have a bar chart that shows you how many orders were made at your store per hour.
++
+[role="screenshot"]
+image::images/lens_advanced_1_2.png[Orders per day]
+
+. Click *Save and return*.
+
+[discrete]
+[[view-the-cumulative-number-of-products-sold-over-time]]
+=== View the cumulative number of products sold on weekends
```

Comment:
This section of the tutorial doesn't contain any recommendations for _when_ to do this. We don't typically recommend using the Filters agg for a single filter, since the KQL bar does the same thing. But the distinction between all of the different filters is pretty confusing to users, and I think we need to explain _why_ here.